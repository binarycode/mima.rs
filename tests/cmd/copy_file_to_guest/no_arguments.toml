args = 'copy-file-to-guest'
status.code = 2
fs.sandbox = true
