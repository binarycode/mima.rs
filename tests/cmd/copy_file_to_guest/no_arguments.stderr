error: The following required arguments were not provided:
    <GUEST_ID>
    <PATH>

USAGE:
    mima copy-file-to-guest <GUEST_ID> <PATH> [FILE_NAME]

For more information try --help
